docs/docs/get-started.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: get-started
+title: Get started
+sidebar_label: Get started
+---
+
+
+## New to Ory ?
+
+- [Ory Projects Overview.](/ecosystem/projects.md)
+What is Ory ? Learn about the open source projects at the heart of Ory:
+- [Ory Community Overview.](/ecosystem/community.md)
+A place where developers share and build: Learn more about the Ory Community:
+- [Ory Kratos Quickstart.](/kratos/quickstart)
+User login and registration for web, mobile, native, or single-page applications.
+- [Ory Keto Quickstart.](/keto/quickstart)
+Complex consistent permission and authorization management.
+- [Ory Hydra Quickstart.](/hydra/5min-tutorial)
+Authenticate third-party users and secure access to your applications and APIs.
+- [Ory Kratos Quickstart.](/oathkeeper/quickstart)
```

Comment:
These links won't work because they will end up in: `https://www.ory.sh/oathkeeper/quickstart`. I'd just link the full URL here, so `https://www.ory.sh/oathkeeper/docs/quickstart`

Reply:
Oh I thought it would work bc it resolves to http://localhost:3000/docs/kratos/quickstart.
Changing now